go.mod

```diff
@@ -6,5 +6,11 @@ require (
 	github.com/aws/aws-lambda-go v1.20.0
 	github.com/aws/aws-sdk-go v1.35.22
 	github.com/google/uuid v1.1.2
+	github.com/kr/pretty v0.1.0 // indirect
+	github.com/stretchr/objx v0.1.1 // indirect
 	github.com/stretchr/testify v1.6.1
+	golang.org/x/net v0.0.0-20200625001655-4c5254603344 // indirect
+	golang.org/x/text v0.3.2 // indirect
+	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
+	gopkg.in/yaml.v2 v2.3.0 // indirect
```

Comment:
Do you know why this has changed? You don't seem to have added any new imports

Reply:
I ran `go mod tidy` - might have done something

Reply:
I guess it won't be a problem?